database/dbGenerator.js

```diff
@@ -1,26 +1,12 @@
+const _ = require('lodash');
 const faker = require('faker');
-const { urlPrefix } = require('./imageUrls');
-const { japanUrls } = require('./imageUrls');
-const { italyUrls } = require('./imageUrls');
-const { usUrls } = require('./imageUrls');
-const { singaporeUrls } = require('./imageUrls');
-const { bathroomUrls } = require('./imageUrls');
-const { bedroomUrls } = require('./imageUrls');
-const { kitchenUrls } = require('./imageUrls');
-const { livingRoomUrls } = require('./imageUrls');
-const { otherUrls } = require('./imageUrls');
+const imageUrls = require('./imageUrls');
```

Comment:
@aku2nnya Another alternative here would be to use de-structuring with commas to get all the variables you need directly (with this example, I think your approach is cleaner, but with 2-3 examples, it can be nifty to do { italyUrls, japanUrls, etc. }).